packages/react-components/src/menu/src/Menu.tsx

```diff
@@ -174,55 +187,55 @@ export function InnerMenu({
     const renderOption = ({
         key,
         index,
-        elementType: ElementType = MenuItem,
+        elementType: As = MenuItem,
         ref,
         content,
         props,
         tooltip
-    }) => (
-        <ElementType
+    }: Omit<CollectionItem, "type">) => (
```

Comment:
Wondering why in this case you choose to use "Omit" instead of "Partial" ?

Reply:
done